### PR TITLE
Add tiny-date-picker w/ npm auto-update

### PR DESCRIPTION
**Pull request by pvnr0082t**
Pull request for issue: #10199
Related issue(s): #10248 

@extend1994 @maruilian11 @cdnjs/intern2 Please help me review this PR, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/chrisdavies/tiny-date-picker
 * NPM package url (optional): https://www.npmjs.com/package/tiny-date-picker
 * License and its reference: MIT, https://github.com/chrisdavies/tiny-date-picker#license-mit
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 6
   - Count of stars: 70
   - Count of forks: 15
 * NPM download stats (optional):
   - Downloads in the last day: 2
   - Downloads in the last week: 394
   - Downloads in the last month: 887

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by pvnr0082t** [comment]
@maruilian11  You're right. I have modified it. Please help me review it again. Thank you.

File: ajax/libs/tiny-date-picker/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "type": "git",
+    "url": "git+https://github.com/chrisdavies/tiny-date-picker.git"
+  },
+  "keywords": [
+    "date",
+    "picker",
+    "chooser",
+    "calendar"
+  ],
+  "author": "Chris Davies <ferociousturtle@gmail.com>",
```
Comment (extend1994) [inline ajax/libs/tiny-date-picker/package.json]: LGTM.
Just want to ask if
```
"author": "Chris Davies <ferociousturtle@gmail.com>"
```

and
```
"author": {
  "name": "Chris Davies",
  "email": "ferociousturtle@gmail.com"
}
```

are both OK?

Comment (maruilian11) [inline ajax/libs/tiny-date-picker/package.json]: It's both OK, but it's better to follow the upstream.

Comment (extend1994) [inline ajax/libs/tiny-date-picker/package.json]: @maruilian11 Thanks for response :+1:

**Comment by maruilian11** [comment]
@PeterDaveHello  Both @extend1994 and I think this PR is fine. And I have help rebase this branch with the latest master branch.

File: ajax/libs/tiny-date-picker/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/tiny-date-picker/package.json]: We don't need to spend too much time on author section actually.

**Pull request merged**